<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow_UserSelectedBed</class>
 <widget class="QMainWindow" name="MainWindow_UserSelectedBed">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1694</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:#222222;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1691</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>0</y>
      <width>331</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/images/logoGIBIC.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:black;
color:white;</string>
    </property>
    <property name="text">
     <string>Connection</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_stadisticWindow">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>300</y>
      <width>651</width>
      <height>591</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/images/sensor1.jpeg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>341</width>
      <height>591</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/images/pacienteDeFrente2.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(0, 128, 64);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_historicRegisters">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>890</y>
      <width>1301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/images/historial4.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>240</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 25 18pt &quot;Helvetica Neue&quot;;
qproperty-alignment: AlignCenter; 
background:rgb(0, 64, 128);</string>
    </property>
    <property name="text">
     <string>Breathing frequency: 18 rpm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>90</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 25 18pt &quot;Helvetica Neue&quot;;
qproperty-alignment: AlignCenter; 
background:rgb(0, 64, 128);</string>
    </property>
    <property name="text">
     <string>Tilt level: 45</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>130</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/images/BedMonitoreoV2.00033.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>90</y>
      <width>321</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/images/controlActivo1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_fourBedsImage">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>471</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/images/UCI3.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>90</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#222222;
color:white;</string>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPressureButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>90</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#222222;
color:white;</string>
    </property>
    <property name="text">
     <string>Distribución</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStadistics">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>90</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#222222;
color:white;</string>
    </property>
    <property name="text">
     <string>Estadísticas</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_stadisticWindow2">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>300</y>
      <width>651</width>
      <height>591</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/images/sensor2.jpeg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonExpositionTimes">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>90</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#222222;
color:white;</string>
    </property>
    <property name="text">
     <string>Tiempos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStadistics_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>90</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#222222;
color:white;</string>
    </property>
    <property name="text">
     <string>Estadísticas1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_zona1_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>310</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
qproperty-alignment: AlignCenter;
font: 25 18pt &quot;Helvetica Neue&quot;; 
border-radius:15;</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_zona1_6">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>310</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
qproperty-alignment: AlignCenter;
font: 25 18pt &quot;Helvetica Neue&quot;; 
border-radius:15;
</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_zona1_9">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>310</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
qproperty-alignment: AlignCenter;
font: 25 18pt &quot;Helvetica Neue&quot;; 
border-radius:15;
</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_zona2_3">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>310</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
qproperty-alignment: AlignCenter;
font: 25 18pt &quot;Helvetica Neue&quot;; 
border-radius:15;
</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_zona2_6">
    <property name="geometry">
     <rect>
      <x>1360</x>
      <y>310</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
qproperty-alignment: AlignCenter;
font: 25 18pt &quot;Helvetica Neue&quot;; 
border-radius:15;
</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_zona1_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>510</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
qproperty-alignment: AlignCenter;
font: 25 18pt &quot;Helvetica Neue&quot;; 
border-radius:15;
</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_zona1_5">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>510</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
qproperty-alignment: AlignCenter;
font: 25 18pt &quot;Helvetica Neue&quot;; 
border-radius:15;
</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_zona1_8">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>510</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
qproperty-alignment: AlignCenter;
font: 25 18pt &quot;Helvetica Neue&quot;; 
border-radius:15;
</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_zona2_2">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>510</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
qproperty-alignment: AlignCenter;
font: 25 18pt &quot;Helvetica Neue&quot;; 
border-radius:15;
</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_zona2_5">
    <property name="geometry">
     <rect>
      <x>1360</x>
      <y>510</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
qproperty-alignment: AlignCenter;
font: 25 18pt &quot;Helvetica Neue&quot;; 
border-radius:15;
</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_zona2_9">
    <property name="geometry">
     <rect>
      <x>1570</x>
      <y>310</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
qproperty-alignment: AlignCenter;
font: 25 18pt &quot;Helvetica Neue&quot;; 
border-radius:15;
</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_zona2_8">
    <property name="geometry">
     <rect>
      <x>1570</x>
      <y>510</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
qproperty-alignment: AlignCenter;
font: 25 18pt &quot;Helvetica Neue&quot;; 
border-radius:15;
</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_zona1_1">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>710</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
qproperty-alignment: AlignCenter;
font: 25 18pt &quot;Helvetica Neue&quot;; 
border-radius:15;
</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_zona1_4">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>710</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
qproperty-alignment: AlignCenter;
font: 25 18pt &quot;Helvetica Neue&quot;; 
border-radius:15;
</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_zona1_7">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>710</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
qproperty-alignment: AlignCenter;
font: 25 18pt &quot;Helvetica Neue&quot;; 
border-radius:15;
</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_zona2_1">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>710</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
qproperty-alignment: AlignCenter;
font: 25 18pt &quot;Helvetica Neue&quot;; 
border-radius:15;
</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_zona2_4">
    <property name="geometry">
     <rect>
      <x>1360</x>
      <y>710</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
qproperty-alignment: AlignCenter;
font: 25 18pt &quot;Helvetica Neue&quot;; 
border-radius:15;
</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_zona2_7">
    <property name="geometry">
     <rect>
      <x>1570</x>
      <y>710</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
qproperty-alignment: AlignCenter;
font: 25 18pt &quot;Helvetica Neue&quot;; 
border-radius:15;
</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_HorizontalDivision1">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>496</y>
      <width>1301</width>
      <height>1</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_HorizontalDivision2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>693</y>
      <width>1301</width>
      <height>1</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>586</x>
      <y>300</y>
      <width>1</width>
      <height>591</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>803</x>
      <y>300</y>
      <width>1</width>
      <height>591</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>300</y>
      <width>1</width>
      <height>591</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>1236</x>
      <y>300</y>
      <width>1</width>
      <height>591</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>1453</x>
      <y>300</y>
      <width>1</width>
      <height>591</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1694</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
